.gitignore

```diff
@@ -9,3 +9,5 @@ dist/
 .tox/
 tags
 .env/
+.tags
```

Comment:
I would have to change all my other repos... At the end of the day it is two more file names in the .gitignore.